osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -50,6 +50,7 @@ public abstract class DrawableRuleset<TObject> : DrawableRuleset, IProvideCursor
         /// <summary>
         /// The key conversion input manager for this DrawableRuleset.
         /// </summary>
+        [Cached]
```

Comment:
this cache has no matching `[Resolved]` attribute and therefore does nothing.

Reply:
I forgot to revert this. Thanks